.github/workflows/pytest_tests_collection.yaml

```diff
@@ -0,0 +1,46 @@
+name: Tests collection
+
+on:
+  pull_request:
+    paths:
+      - tests/**
+
+jobs:
+  test-collection:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: pip install -r requirements.txt
+
+      - name: Install dependencies
+        run: python3 setup.py install
+        working-directory: deps/wazuh_testing
```

Comment:
If those steps are different please specify a different name. I recommend:

- Install framework dependencies
- Install the QA framework

Reply:
Done 1f5ab15